package.json

```diff
@@ -14,53 +13,61 @@
     "playground": "concurrently \"yarn:build-bundle --watch\" \"yarn:playground:build-app --watch\" \"yarn:playground:build-server --watch\" \"node playground/server.js\"",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:bundle": "rollup -c .storybook/rollup.config.js",
-    "test:unit": "vue-cli-service test:unit",
-    "tdd": "vue-cli-service test:unit --watch"
+    "test:unit": "rollup -c tests/rollup.config.test.js && karma start --single-run",
```

Comment:
I don't get why it's necessary to execute rollup and then karma.
Can't just karma build the test files using the rollup-preprocessor?

Reply:
This was done in order to handle multiple entry points. I've changed that by introducing a single entry point to build the test suite, as you suggested orally, thanks!